languageserver/server/server.go

```diff
@@ -1120,6 +1120,7 @@ func (s *Server) getDiagnostics(
 		runtime.FileLocation(uri),
 		sema.WithPredeclaredValues(valueDeclarations),
 		sema.WithPredeclaredTypes(typeDeclarations),
+		sema.WithOriginsAndOccurrencesEnabled(true),
```

Comment:
It's default off, and only turned on for language server?

Reply:
Correct, it's disabled by default, because it's not needed to determine all semantic errors.
The occurrences and origin functionality is only used by the language server to e.g. query type information for a specific location in the source code.